### DISCUSSION TITLE
Add custom lints

### DISCUSSION THREAD

**Discussion by mehdihadeli** [golangci/golangci-lint#3395] [Q&A] [answered]
Could we add some custom none-exsting lints?

**Comment by bombsimon** [comment, marked as answer]
You mean like this: [How to add a private linter to `golangci-lint`](https://golangci-lint.run/contributing/new-linters/#how-to-add-a-private-linter-to-golangci-lint)?

**Reply by mehdihadeli** [reply]
Yes, Thanks
